backend/src/main/kotlin/cz/loono/backend/db/model/HealthcareProvider.kt

```diff
@@ -0,0 +1,158 @@
+package cz.loono.backend.db.model
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.IdClass
+import javax.persistence.JoinColumn
+import javax.persistence.JoinTable
+import javax.persistence.ManyToMany
+import javax.persistence.Table
+
+@Entity
+@IdClass(HealthcareProviderId::class)
+@Table(name = "\"healthcare_provider\"")
+data class HealthcareProvider(
+
+    @Id
+    @Column(name = "location_id", nullable = false, columnDefinition = "TEXT")
+    val locationId: String = "",
+
+    @Id
+    @Column(name = "institution_id", nullable = false, columnDefinition = "TEXT")
+    val institutionId: String = "",
```

Comment:
The H2 database fell on its face in tests the last time I tried non-integer primary keys. I am interested in seeing this run.

Reply:
I don't know about any constraint that the primary key has to be integer.